src/TriggerService.Tests/TriggerService.Tests.csproj

```diff
@@ -15,13 +15,14 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.10.2" />
-    <PackageReference Include="Microsoft.Azure.Batch" Version="15.4.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0" />
-    <PackageReference Include="Moq" Version="4.18.4" />
-    <PackageReference Include="MSTest.TestAdapter" Version="3.0.2" />
-    <PackageReference Include="MSTest.TestFramework" Version="3.0.2" />
-    <PackageReference Include="coverlet.collector" Version="3.2.0">
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="8.0.1" />
```

Comment:
I messed up. This added line should have been added to `src/TriggerService.Tests/TriggerService.csproj`.

Sorry